GUI/GUIMod.cs

```diff
@@ -186,8 +199,9 @@ public GUIMod(string identifier, IRegistryQuerier registry, KspVersionCriteria c
             }
 
             // If we have a homepage provided, use that; otherwise use the spacedock page, curse page or the github repo so that users have somewhere to get more info than just the abstract.
-
             Homepage = "N/A";
+
+            SearchableIdentifier = CkanModule.nonAlphaNums.Replace(Identifier, "");
```

Comment:
Why ware we calculating this one differently? Can't we just set it to `mod.SearchableIdentifier`?